images/update-dbg/entrypoint.sh

```diff
@@ -30,10 +30,11 @@ export GIT_AUTHOR_NAME=${BOT_NAME}
 export GIT_AUTHOR_EMAIL=${BOT_MAIL}
 
 # Update the DBG from crawling results.
-# $2: chart name
 generate_driverkit_configs() {
-    make -C driverkit generate/auto
-    return $?
+    dbg-go configs cleanup -a amd64
```

Comment:
@FedeDP why not doing for all architectures? Let's imagine we increase the architecture support

Reply:
Because `dbg-go` takes the architecture as input parameter, and does not support looping on architectures (that's because it needs to know that target arch while generating or building dbg configs).

Reply:
No problem. Just my curiosity